<?xml version="1.0"?>
<launch>
  <arg name="config_path"       default = "/config/ct_odometry_r3live.yaml" />
  <!-- <arg name="config_path"       default = "/config/ct_odometry_fastlivo.yaml" /> -->
  <!-- <arg name="config_path"       default = "/config/ct_odometry_viral.yaml" /> -->
  <!-- <arg name="config_path"       default = "/config/ct_odometry_lvisam.yaml" /> -->
  <arg name="bag_path"          default = "" />  
  <arg name="pasue_time"        default = "-1" />
  <arg name="verbose"           default = "false" />  
  

  <!-- <node pkg="cocolic" type="odometry_node" name="$(anon odometry_node)" output="screen" clear_params="true" 
        launch-prefix="gdb -ex run &#45;&#45;args"> -->
  <node pkg="cocolic" type="odometry_node" name="$(anon odometry_node)" output="screen">
    <param name="config_path"     type="string" value="$(find cocolic)$(arg config_path)" />
    <!-- if 'bag_path' is none here then use get the 'bag_path' params in the yaml file-->
    <param name="bag_path"        type="string" value="$(arg bag_path)" />
    <param name="pasue_time"      type="double" value="$(arg pasue_time)" />
    <param name="verbose"         type="bool"   value="$(arg verbose)" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz_odom" output="log" required = "true" args="-d $(find cocolic)/config/coco.rviz" />
</launch>
